SpecificTarget.Build.props

```diff
@@ -0,0 +1,22 @@
+<Project>
+  <PropertyGroup>
+    <!-- Setting any of these to true will cause all the other platforms to get excluded.
+        Cake writes out to these properties when you've used a specific target switch
+    -->
+    <_IncludeWindows></_IncludeWindows>
+    <_IncludeTizen></_IncludeTizen> 
+    <_IncludeAndroid></_IncludeAndroid> 
+    <_IncludeIos></_IncludeIos> 
+    <_IncludeMacCatalyst></_IncludeMacCatalyst> 
+  </PropertyGroup>
```

Comment:
One pattern we did here is make a `Configuration.Override.props.in` that you can manually copy and edit locally:

* https://github.com/xamarin/xamarin-android/blob/main/Configuration.Override.props.in

Then import `Configuration.Override.props` if it exists:

```xml
<Import
    Project="Configuration.Override.props"
    Condition="Exists('Configuration.Override.props')"
/>
```

And then add `Configuration.Override.props`  to the `.gitignore`. Locally you can create this file and uncomment/comment things.

Maybe dotnet/maui could do a `Directory.Build.Override.props.in`? I think our example was made before `Directory.Build.props/targets` was a feature of MSBuild.

Reply:
I like that name better too `Directory.Build.Override.props.in`

Changes pushed up.